server/api/services/events.service.js

```diff
@@ -153,14 +153,18 @@ class EventsService {
       const points = {};
 
       winners.forEach((winner, index) => {
-        points[`score.${eventData.category}`] = eventData.importance * 5 + (4 - index) * 5;
-        points['totalScore'] = eventData.importance * 5 + (4 - index) * 5;
-
-        const achievement = {};
-        if (index === 0) achievement['achievements.first'] = eventData.name;
-        else if (index === 1) achievement['achievements.second'] = eventData.name;
-        else if (index === 2) achievement['achievements.third'] = eventData.name;
-
+        points[`score.${eventData.category}`] = eventData.importance * 5 + (2 - index) * 5;
+        points['totalScore'] = eventData.importance * 5 + (2 - index) * 5;
+
+        winner.forEach(userId => {
+          promises.push(
+            achievementModel.create({
```

Comment:
The achievement model has not been imported in this file. Also, please rename the model file to `achievement` instead of `achievements` because it's a model and not a query result.